### PR TITLE
Marketingpage 2.0

### PR DESCRIPTION
**Pull request by lukasbuenger**
image

image

I still expect text changes to happen, but as far as logic and UX is concerned, this is pretty much it.

### CODE REVIEW

File: components/Marketing/PreviewFront.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+const PreviewFront = ({ url, data, data: { front }, t, me, isMember }) => {
+  return (
+    <Fragment>
+      <Prestitial isMember={isMember} me={me} t={t} />
+      <Loader loading={data.loading} error={data.error} message={t('pages/magazine/title')} render={() => {
+        if (!front) {
+          return <StatusError
+            url={url}
+            statusCode={404}
+            serverContext={this.props.serverContext} />
```
Comment (tpreusse) [inline components/Marketing/PreviewFront.js]: there is no this in a functional component

**Comment by lukasbuenger** [comment]
@tpreusse:

I forgot to feedback on the translations issue. I didn't feel too comfortable removing the old keys before everything is kind of set in stone. If we had to pull translations tomorrow for an unrelated critical bug fix or wording issue, we'd be screwed. 
It was clear to me that I would clean the translations as soon as we have the final copies.

**Comment by tpreusse** [comment]
@lukasbuenger re `translation.json`: yeah the gsheet doesn't synchronously branch. But ultimately it's committed on branches and also historized on the google side. I wouldn't worry too much, for small changes it's also easy to directly edit the json file or manually cherry pick it together.

File: components/Marketing/Preview.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    {!me && <Frame raw url={url} meta={meta}>
+      <Container style={{ maxWidth: '665px' }}>
+        <SignUp />
+      </Container>
+    </Frame>
+    }
+    {me && <Front
+      url={url}
+      beforeNote={
+        <Container style={{ maxWidth: '100%' }}>
```
Comment (tpreusse) [inline components/Marketing/Preview.js]: Correct order
```js
import Box from '../Frame/Box'

(
    <Box>
      <Container>
        <Interaction.P>
        {children}
        </Interaction.P>
      </Container>
    </Box>
)
```

**Pull request merged**